### DISCUSSION TITLE
[インポスター/Impostor] 未定(会議をダイレクトに封じる)

### DISCUSSION THREAD

**Discussion by yukieiji** [yukieiji/ExtremeRoles#293] [Ideas]
- アイディアが完全に固まりきってないので意見求めも含めてディスカッション化
  - これらの能力、仕様は変更されることがあります

## 能力詳細
- 名前：未定
- 色：インポスターレッド/Impostor
- 陣営：インポスター

- 役職コンセプト：会議時間や会議クールをリセットすることで会議をダイレクトに封じる
  - 定期的にボタンを押すことでインポスターの邪魔をすることでキルできなくなるのでそれに対抗できる役職として設計したい

- ボタン能力：「？？？」
  - 死体に対して数秒間止まることで発動
  - 発動が完了すると会議クールリセット + 「刻のかけら」の設置をするであろう判定が入る
  - 会議クールを一定数永続減少
  - 死体の絵を変更
- 回収アイテム「刻のかけら」
  - 会議時間が封じ込まれたかけら、回収することで会議時間が増える
  - 会議終了後に特定の箇所に設置される
  - 設定としてボタン能力1回の発動で30秒減る + 設置個数6個の場合 => 一個回収5秒増える

## 未検討事項
- ただ単に発動するボタン能力であるとコストが軽すぎる、現状だと回収される「刻のかけら」に対してコストが重すぎると思われる
  - TOHのタイムシーフ見たくキルした瞬間でもいいが分かりにくすぎる

**Comment by flog4624** [comment]
実は会議を封じる役職は以前から考えておりました。
ここに自分の考えた役職の案を書いておきます。

役職名:ジャミンガー
緊急会議の発動を封じるインポスター陣営

・一定回数キルするごとに「妨害電波」能力を1回発動できるようになる。この能力はストックされない。
妨害電波能力ボタンを押すことで妨害電波を放ち、味方や自身を含めて生きているプレイヤー全員を一定時間、封印状態にする。
封印状態のプレイヤーは死体通報・緊急会議ボタンが封じられる。


オプション
妨害電波に必要なキル数
妨害電波の最大使用可能回数
封印状態の持続時間


長所
・緊急会議を防いで安全にダブルキル・トリプルキルができる

短所
・吊り回避のためには姑息な手段（本来の意味）に過ぎないため現場を押さえられた場合、勝ちきれないと結局吊られてしまう
・封印状態の間はタスクが中断されないのでタスク進行に対しては無力


他陣営への影響
クルー陣営:封印状態になった場合はできるだけ複数人で動き、直後の会議で情報をまとめるとよい。また、カースメーカーは封印状態だと能力使用現場を見られても通報されないので動きやすくなる
ニュートラル陣営:封印状態中に動きやすくなる
